JavaFXUserInterface/src/main/resources/AssetInfo.fxml

```diff
@@ -1,92 +1,143 @@
-                                                        <TextField fx:id="siteOutput" layoutX="120.0" layoutY="186.0" prefWidth="200.0" text="T" />
-                                                        <TextArea fx:id="descriptionOutput" layoutX="415.0" layoutY="105.0" prefHeight="52.0" prefWidth="250.0" text="T" />
-                                                        <Text fill="#828282" layoutX="15.0" layoutY="87.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Model:" textAlignment="RIGHT" wrappingWidth="90.0" />
-                                                        <Text fill="#828282" layoutX="15.0" layoutY="116.0" strokeType="OUTSIDE" strokeWidth="0.0" text="RUL:" textAlignment="RIGHT" wrappingWidth="90.0" />
-                                                        <Text fx:id="modelOutput" layoutX="129.0" layoutY="87.0" strokeType="OUTSIDE" strokeWidth="0.0" text="T" />
-                                                        <Text fx:id="rulOutput" layoutX="129.0" layoutY="116.0" strokeType="OUTSIDE" strokeWidth="0.0" text="T" />
-                                                        <Text fill="#828282" layoutX="312.0" layoutY="87.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Category:" textAlignment="RIGHT" wrappingWidth="90.0" />
-                                                        <TextField fx:id="categoryOutput" layoutX="415.0" layoutY="70.0" prefHeight="25.0" prefWidth="200.0" text="T" />
-                                                        <Text fx:id="assetNameLabel" fill="#828282" layoutX="312.0" layoutY="29.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Asset Name:" textAlignment="RIGHT" wrappingWidth="90.0" />
-                                                        <TextField fx:id="assetNameOutput" layoutX="415.0" layoutY="12.0" prefWidth="200.0" text="T" />
-                                                        <Text fill="#828282" layoutX="15.0" layoutY="145.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Status:" textAlignment="RIGHT" wrappingWidth="90.0" />
-                                                        <Text fx:id="recommendationOutput" layoutX="129.0" layoutY="145.0" strokeType="OUTSIDE" strokeWidth="0.0" text="T" />
-                                                        <ImageView fx:id="imageView" fitHeight="150.0" fitWidth="200.0" layoutX="867.0" layoutY="13.0" pickOnBounds="true" preserveRatio="true" />
+                                                        <Text fx:id="assetTypeLabel" fill="#828282" layoutX="15.0"
+                                                              layoutY="58.0" strokeType="OUTSIDE" strokeWidth="0.0"
+                                                              text="Asset Type:" textAlignment="RIGHT"
+                                                              wrappingWidth="90.0"/>
+                                                        <Text fx:id="serialNumber" fill="#828282" layoutX="15.0"
```

Comment:
multiple unresolved fx:id reference in this file as well!

Reply:
done